src/components/Dialogs/ActionsModal.tsx

```diff
@@ -77,6 +77,30 @@ const actions = {
   },
 };
 
+const RenderItem = (item: { name:string, function:string, data?:boolean, runAction:Function }) => {
+  const [showInfo, setShowInfo] = useState<boolean>(false);
+
+  return (
+    <>
+      <div className="flex justify-between font-semibold">
+        <span>{item.name}</span>
+        <div>
+          <span className="inline-block pr-2 text-highlight-1" onClick={() => setShowInfo(!showInfo)}><Icon className="cursor-pointer" path={mdiInformationOutline} size={1}></Icon></span>
+          <span className="inline-block text-highlight-1" onClick={() => item.runAction(item.name, item.function)}><Icon className="cursor-pointer" path={mdiPlayCircleOutline} size={1} /></span>
```

Comment:
Why not just pass a `onClick` prop instead of the `function` and `runAction` props?